interface/new/new_comprehensive_save.php

```diff
@@ -28,11 +28,9 @@
 // other interfaces can still read the data during this lock, however
 // sqlStatement("lock tables patient_data read");
 
-$result = sqlQuery("SELECT MAX(pid)+1 AS pid FROM patient_data");
-
+$result = sqlQuery("SELECT id AS pid FROM patient_data ORDER BY id DESC LIMIT 1");
```

Comment:
should not be using id here. it should be:
``````
$result = sqlQuery("SELECT pid FROM patient_data ORDER BY id DESC LIMIT 1");
``````
(Note you also have 2 unrelated files with changes in this commit; not a biggie, which we can deal with when/if bring in this fix)